CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## 0.21.1 (September 10th, 2024)
```

Comment:
version 0.21.1 does not exist. If these changes are already in master, all of them should be documented under next release which is 0.22.0

Reply:
Also your changes are missing here (adding DN attributes and all of that)